tests/unit/TodoList1.spec.ts

```diff
@@ -2867,11 +2874,15 @@ describe('TodoList - Click Tests - Full and Limited Restoration', () => {
   ]
 
   for (const test of fullAndLtdRestorationTests) {
-    xit(`redirects to Create URL when 'Resume' is clicked on a ${test.restorationType} draft applciation`, async () => {
+    it(`redirects to Create URL when 'Resume' is clicked on a ${test.restorationType} draft applciation`, async () => {
       // init store
       sessionStorage.clear()
-      configurationStore.setTestConfiguration({ configuration: configurationStore.configuration },
-        { key: 'VUE_APP_BUSINESS_CREATE_URL', value: 'https://create.url/' })
+      const configuration = {
+        VUE_APP_BUSINESS_CREATE_URL: 'https://create.url/'
+      }
```

Comment:
Could you set this in the beforeAll() above?

(Do it once instead of every iteration.)

Reply:
Fixed, thanks.